code/modules/spells/spell_types/undirected/conjure_item/smoke_bomb.dm

```diff
@@ -0,0 +1,16 @@
+/datum/action/cooldown/spell/undirected/conjure_item/smoke_bomb
```

Comment:
Suggesting the same with what I said about the garotte spell, especially since smoke bombs are craftable.